common/state/state_kv.cpp

```diff
@@ -1109,6 +1576,7 @@ void pstate::data_node_io::cache_modified(unsigned int block_num)
 void pdo::state::block_warehouse::serialize_block_ids(pdo::state::StateNode& node)
 {
     node.ClearChildren();
+    SAFE_LOG(PDO_LOG_ERROR, "state blocks: %u", blockIds_.size());
     for (unsigned int i = 0; i < blockIds_.size(); i++)
```

Comment:
this is an error level debugging statement? please remove!

Reply:
done